src/en/space-station-14/departments/security/proposals/reduced-metagaming.md

```diff
@@ -0,0 +1,234 @@
+
+The Uplink Patch would turn any object into a possible Uplink, making the permutations for what to confiscate anything in the Syndie's inventory. Even if a SecOff would be overzealous and try to take everything, the uplink could be an implant as well. With the changes in the Implant Problem, this would require Security to have strong suspicion of where the uplink would be to actually catch it.
+
+## The Thief Gloves Problem
+
+```admonish quote
+Checking black gloves without proper evidence leading to them
+```
+```admonish quote
+Checking for Chameleon gear when there is no reason to suspect it feels kinda metagaming, yeah? Especially since you now have to put the items on to even check.
+```
+
+Thief gloves come in two varieties, each with their own problem. 
+
+1. Normal Thief Gloves are indistuingishable from black gloves. The only way to know they have been used is if someone finds out an item on their person has been stolen, and even then Security are not allowed to know they must search for black gloves unless an uplink has been previously found. They also stand out for crew who don't normally wear black gloves, making overzealous SecOffs search and possibly even confiscate them.
+
+2. Chameleon Thief Gloves mean to remedy this. However, as they leave holographic fibers on everything they touch it becomes very obvious whenever they have been used. It leads to a rather opaque and unintuitive game of trying to not wear the thief gloves except in the most critical of moments, making finding a spare pair of gloves in someone's bag highly suspicious. They're also trivial to check since it's just to put them on and check whether they allow Chameleon properties.
+
+### Proposed Solution
+
+- Thieving Gloves and Chameleon Thieving Gloves are merged into a single item: The Thief Glovebox.
+  - When purchased/obtained, the user is given a box. Opening this box activates a UI similar to a Chameleon UI.
+  - Once a glove design is chosen, the user is given a pair of thieving gloves with that design.
+  - The gloves give off matching fibers. They are not Chameleon.
+- Non-engineering insulated gloves/budget insulated gloves now come in 3 different colors with corresponding colored fibers. They are still called "Insulated Gloves"/"Budget Insulated Gloves". 
+- Rule change: Security has knowledge of Thieving Gloves and may confiscate the gloves of anyone who has stolen or attempted to steal contraband.
+- Rule change: An additional rule is added: Security may *not* test Thieving Gloves by putting them on and trying to steal an item off someone.
+
+This means that a thief is safe from having their gloves lost before committing a crime. Detectives can no longer use holofibers to confirm a Thief antag, but must instead rely on the fibers of the gloves around the crime scene. The addition of additional insuls colors means selecting yellow insulated gloves is no longer a safe option to blend in with all the other yellow gloves.
```

Comment:
I genuinely just don't like this, so this is much more of a personal opinion than a design opinion, but never-the-less here's some opinions for you:
- I think the real issue is that glove colors other than yellow are extremely rare, with little purpose. Some SS13 servers have solved this by making the other gloves actually do things (i.e. sterile gloves are sterile, normal black gloves are *heat resistant*, etc.)
  - Implementing the black gloves heat resistant mechanic and removing the ability for other gloves to do things like pull lights would be a good first step in normalizing them.
  - Thieves gloves are intended to be a stealth mechanic, end of the day, so it is *imo* better to address the things that make them not stealthy instead of making them even stealthier.
- Chameleon gloves work fine as is, especially if the above proposed changes are made to make having multiple glove pairs more common. I don't think we need multiple insul colors to remedy this, just reason to use multiple gloves for different purposes.

Reply:
I think that your approach is fully valid, just maybe harder to balance for as there's a risk one kind of gloves is always the "meta". 

The reason why I'm for removing the Chameleon Thief gloves (and its corresponding fibers) is because if Security are allowed knowledge of how Thieving Gloves work, it becomes trivial to know they are in play the moment the victim says "My X was stolen off me". However making the gloves a single type (like the normal gloves) have its own issues as stated in the proposal, so the glovebox is a middle ground.

Ultimately the goal is glove diversity. Whether that's achieved via making more gloves viable, or making the viable gloves more varied, either works for me. :)

Reply:
I will note, I think the thieving gloves themselves are a mis-executed stealth item, and I'm very unsure on how to actually fix them because their specific usecase (theft without an indicator) is too specific to actually *be* a good stealth item.

Reply:
As someone who is a prolific user of the thief gloves as both thief and traitor, the gloves are incredibly effective when used in conjunction with chaos. The stealth usefulness they provide is for the moments before and during the theft; once you've absconded and the victim realizes they've lost their stuff it's important to hide the gloves as they do not stand up to targetted inspection. 

Considering the alternative is that you need to kill your target to steal from them I think they are in a good place when it comes to functionality.